.github/workflows/publish.yml

```diff
@@ -0,0 +1,100 @@
+name: Publish Package
+
+on:
+  push:
+    branches:
+      - test_pypi_release
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
maybe let's use 3.12 or 3.13?
Then we can run UTs for python matrix for 3.10 3.11 3.12 3.13 will create issue for that :)